lib/api/tests/externals.rs

```diff
@@ -377,3 +377,33 @@ fn function_outlives_instance() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn manually_call_function() -> Result<()> {
```

Comment:
Awesome! This is a great test, although the name of the test is not super clear.

Can we name it `manually_generate_wasmer_env` instead ?